### ISSUE TITLE
`withCopyFileToContainer` does not work, but `copyFileToContainer` does

### ISSUE THREAD

**Issue by rocketraman** [testcontainers/testcontainers-java#1183]
Using 1.10.5, using :

```kotlin

    CONTAINER
    // none of these work
//      .withClasspathResourceMapping("foo.png", "/foo.png", BindMode.READ_WRITE)
//      .withCopyFileToContainer(MountableFile.forClasspathResource("foo.png"), "/foo.png")
//      .withCopyFileToContainer(MountableFile.forHostPath("/full/path/to/foo.png"), "/foo.png")
//      .withFileSystemBind("/full/path/to", "/to", BindMode.READ_ONLY)
      .withCommand("/bin/sh", "-c", "trap : TERM INT; sleep infinity & wait")
      .start()
```

None of the above commented out mechanisms seem to work -- there are no errors, but the file is simply not present in the container.

However, the corresponding copyFileToContainer commands do seem to work:

```kotlin
    CONTAINER
      .withCommand("/bin/sh", "-c", "trap : TERM INT; sleep infinity & wait")
      .start()

    // works
    CONTAINER.copyFileToContainer(MountableFile.forClasspathResource("foo.png"), "/foo.png")

    // also works
    CONTAINER.copyFileToContainer(MountableFile.forHostPath("/full/path/to/foo.png"), "/foo.png")

    // but still does not work
    CONTAINER.addFileSystemBind("/full/path/to", "/to", BindMode.READ_ONLY)
```

Debug output:

```
2019-01-25 15:24:26,101 INFO  --- [                main] org.tes.doc.DockerClientProviderStrategy          : Found Docker environment with Environment variables, system properties and defaults. Resolved: 
    dockerHost=unix:///var/run/docker.sock
    apiVersion='{UNKNOWN_VERSION}'
    registryUrl='https://index.docker.io/v1/'
    registryUsername='raman'
    registryPassword='null'
    registryEmail='null'
    dockerConfig='DefaultDockerClientConfig[dockerHost=unix:///var/run/docker.sock,registryUsername=raman,registryPassword=<null>,registryEmail=<null>,registryUrl=https://index.docker.io/v1/,dockerConfigPath=/home/raman/.docker,sslConfig=<null>,apiVersion={UNKNOWN_VERSION},dockerConfig=<null>]'
2019-01-25 15:24:26,286 INFO  --- [                main] org.tes.DockerClientFactory                       : Connected to docker: 
  Server Version: 1.13.1
  API Version: 1.26
  Operating System: Fedora 29 (Twenty Nine)
  Total Memory: 64339 MB
...
2019-01-25 15:24:28,396 INFO  --- [                main] org.tes.DockerClientFactory                       : Ryuk started - will monitor and terminate Testcontainers containers on JVM exit
        ℹ︎ Checking the system...
        ✔ Docker version should be at least 1.6.0
...
        ✔ Docker environment should have more than 2GB free disk space
```

**Comment by bsideup** [comment]
Hi @rocketraman,
Do you think you can provide a reproducible example? Because all these methods are tested.
Also, I can't see "file is mountable" check in the output. Did you turn off the checks?

**Comment by rocketraman** [comment]
> Also, I can't see "file is mountable" check in the output. Did you turn off the checks?

Odd, I didn't turn off the checks (you saw the other two checks), but I only see the two checks I showed above. Not sure why the other checks are not listed.

Also odd is that I just re-ran this, and now the `withClasspathResourceMapping` and `withCopyFileToContainer` methods *are* working. Weird, I was sure I was doing the same code before, but perhaps not.

I still can't do `withFileSystemBind` though.

The reproduction recipe is basically what I've shown above. The only thing I've omitted is the container I am using (a container based on `openjdk:11`), the fact that I use Kotlin (I don't see how that would make a difference).
